libraries/tock-register-interface/src/lib.rs

```diff
@@ -53,7 +53,8 @@
 //! ------
 //! - Shane Leonard <shanel@stanford.edu>
 
-#![no_std]
+//#![no_std]
+#![cfg_attr(not(feature = "expectations"), no_std)]
```

Comment:
Purely-subjective style question: Maybe it would be better to leave `#![no_std]` in place and instead place `extern crate std;` in `expectations.rs`?

We would not have the prelude in `expectations.rs`, so e.g. we would need to manually add `use std::vec::Vec;`, but maybe that's actually better?

Reply:
Done.